app/src/androidTest/java/ch/epfl/qedit/quiz/QuizFragmentsTestUsingDB.java

```diff
@@ -26,7 +28,17 @@ public QuizViewModel setup(FragmentTestRule testRule, Fragment fragment) {
         QuizViewModel model =
                 new ViewModelProvider((ViewModelStoreOwner) testRule.getActivity())
                         .get(QuizViewModel.class);
-        model.setQuiz(Util.createMockQuiz("Test"));
+
+        model.setQuiz(
+                new Quiz(
+                        "Test",
+                        Arrays.asList(
+                                new Question(
+                                        "The matches problem",
+                                        "How many matches can fit in a shoe of size 43 ?",
+                                        "matrix3x3"),
+                                new Question(
+                                        "Title 2", "Test answer format", "testAnswerFormat"))));
```

Comment:
mdr code climat ^^ this is ... beatifull...this is art

Reply:
Picasso would be proud of spotless!

Reply:
LMAO, if it was possible to set the number of columns in the code, I would try to reduce it to see if it would produce even more artistic results